integration/scripts/download-previous-release.sh

```diff
@@ -5,8 +5,7 @@ GIT_REPO_URL="git@github.com:weaveworks/eksctl"
 RELEASE_URL_FORMAT="https://github.com/weaveworks/eksctl/releases/download/%s/eksctl_Linux_amd64.tar.gz"
 
 download_previous_release() {
-    previous_tag=$(git ls-remote --tags $GIT_REPO_URL | egrep -v "latest_release|\-rc|{}" | cut -d/ -f3 | sort -Vr \
-        | tail -n "+${GO_BACK_VERSIONS}" | head -1)
+    previous_tag=$(git ls-remote --tags $GIT_REPO_URL | grep -E -v "latest_release|\-rc|\{\}" | cut -d/ -f3 | sort -Vr | tail -n "+${GO_BACK_VERSIONS}" | head -1)
```

Comment:
Any reason to merge the two lines into one? I think it looked more readable before.

Reply:
oh, yeah, that was just for when I was testing, let me get it back to 2 lines